code/modules/antagonists/eldritch_cult/eldritch_knowledge.dm

```diff
@@ -144,15 +169,13 @@
 	var/list/compiled_list = list()
 
 	for(var/H in GLOB.carbon_list)
-		if(!ishuman(H))
-			continue
```

Comment:
You should restore this line. We don't have `GLOB.human_list`

Reply:
No need; now checks if istype isntead of ishuman, see the lines below.

Reply:
Ohhhhh there it is. I missed that one.